app/merge-translations.py

```diff
@@ -45,13 +45,13 @@ def output_translations(translations, aFile):
     alpha_keys = set(alpha_translations.keys())
     beta_keys = set(beta_translations.keys())
 
-    for aKey in alpha_keys.difference(alpha_keys):
```

Comment:
What is this for?

Reply:
Presumably it prints out any keys that are different between the two. Looks as though it was previously comparing one to itself and was never printing. I don't know why we need to print them though